src/draw_symbols.c

```diff
@@ -3723,8 +3730,22 @@ void Select_symbol( Widget UNUSED(w), XtPointer UNUSED(clientData), XtPointer UN
     XtManageChild(my_form);
     XtManageChild(pane);
 
+   // Resize dialog to exactly fit my_form w/o scrollbars
+    XtVaGetValues(my_form,
+                  XmNwidth, &width,
+                  XmNheight, &height,
+                  NULL);
+    XtVaSetValues(select_symbol_dialog,
+                  XmNwidth, width+10,
+                  XmNheight, height+10,
+                  NULL);
+    if (debug_level & 1)
+    {
+      fprintf(stderr,"Select_symbol dialog size: X:%d\tY:%d\n", width, height);
+    }
+
     XtPopup(select_symbol_dialog,XtGrabNone);
-    fix_dialog_size(select_symbol_dialog);
+    //fix_dialog_size(select_symbol_dialog);
```

Comment:
I am 10000% in agreement with this.  This is a very bad habit and Xastir already has too many lines of commented out stuff that should have been removed instead of being commented out.  Some of it is decades old and actually would break things if uncommented.  Delete, don't comment.

Reply:
I swept through the code and commented that out everywhere, then added it back in for a few dialogs. When I'm all done I'll delete it, but not yet.